src/tests/integration/test_postgreSQL.py

```diff
@@ -0,0 +1,17 @@
+# flake8: noqa
+from fixtures.database import *
```

Comment:
star imports are discouraged

Reply:
Fix in [415a581](https://github.com/ploomber/jupysql/pull/117/commits/415a581b29f8a50bb976efd4ea55efafc147b65a)